### PR TITLE
Feature/sales overview

### PR DESCRIPTION
**Pull request by armen-h**
I implemented the graphical layout of the listing overview. Listed items are static for now.

Still to be done: fetch listings from the database to dynamically construct the list, fetch attributes of the clicked listing to pass them to the SaleDetails class.

### CODE REVIEW

File: app/src/androidTest/java/ch/epfl/polybazaar/SalesOverviewTest.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                    SalesOverview.class,
+                    true,
+                    false);
+
+    @Test
+    public void testOnCreate() {
+        Intent intent = new Intent();
+        Bundle b = new Bundle();
+        b.putString("title1", "Algebre Linéaire by David C. Lay" );
+        b.putString("price1", "CHF 13");
```
Comment (Walterieux) [inline app/src/androidTest/java/ch/epfl/polybazaar/SalesOverviewTest.java]: You can directly use intent.putExtra("title1",...);

File: app/src/androidTest/java/ch/epfl/polybazaar/SalesOverviewTest.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    }
+
+    @Test(expected = NullPointerException.class)
+    public void testOnCreateThrowAnException() {
+        Intent intent = new Intent();
+        Bundle b = new Bundle();
+        activityRule.getActivity().onCreate(b);
+    }
+
+}
```
Comment (Walterieux) [inline app/src/androidTest/java/ch/epfl/polybazaar/SalesOverviewTest.java]: This test doesn't improve coverage

Comment (armen-h) [reply app/src/androidTest/java/ch/epfl/polybazaar/SalesOverviewTest.java]: Added additional test to improve coverage

File: app/src/androidTest/java/ch/epfl/polybazaar/SalesOverviewTest.java [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    public final ActivityTestRule<SalesOverview> activityRule =
+            new ActivityTestRule<>(
+                    SalesOverview.class,
+                    true,
+                    false);
+
+    @Test
+    public void testOnCreate() {
+        Intent intent = new Intent();
+        Bundle b = new Bundle();
```
Comment (JanMaxime) [inline app/src/androidTest/java/ch/epfl/polybazaar/SalesOverviewTest.java]: Could you rename the variable with a more expressive name ?

**Pull request merged**